qemu/tests/balloon_hotplug.py

```diff
@@ -70,6 +87,11 @@ def run_pm_test(pm_test, plug_type):
         vm.params["balloon_dev_devid"] = "balloon%d" % idx
         vm.params["balloon_dev_add_bus"] = "yes"
 
+        if params.get("blnsrv_enable", "no") == "yes":
+            error_context.context("Install and check balloon service in windows"
+                                  "guest", logging.info)
+            enable_balloon_service(session)
```

Comment:
Do we need to close the session if we will not use it anymore?

Reply:
Sure, will modify it.
Thanks

Reply:
Modified.